db/migrate/20220321091220_add_personal_owner_to_teams.rb

```diff
@@ -0,0 +1,19 @@
+class AddPersonalOwnerToTeams < ActiveRecord::Migration[6.1]
+  def up
+    add_column :teams, :type, :string, null: false
+
+    execute "UPDATE teams SET type = 'Team::Shared' WHERE type = ''"
```

Comment:
```suggestion
    execute "UPDATE teams SET type = 'Team::Shared'"
```
do we need WHERE condition here?